src/stylesheets/Login.scss

```diff
@@ -0,0 +1,23 @@
+#login {
+  margin-bottom: 15px;
+}
+
+.row {
+  .input-field {
+    input:focus:not {
+      border-bottom: 1px solid #ccc;
+    }
+    input:focus + input:not.invalid {
+      border-bottom: 1px solid #00bcd4 !important;
+    }
+    input:focus + label {
+      color: #00bcd4 !important;
+    }
+    margin-bottom: -10px;
+  }
+}
+
+.btn {
```

Comment:
avoid making a direct css change to the class btn because it's global so it'll change all of the buttons we declare to have this margin.